app/javascript/react/components/DescriptionField.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+const DescriptionField = props => {
+  return (
+    <label>
+      {props.label}
+      <textarea
+        name="description-field"
+        type="text"
+        value={props.content}
+        onChange={props.handleDescriptionChange}
+      />
+    </label>
+  );
```

Comment:
hmmm I dont think the label tag is supposed to surround the textarea tag, but instead be adjacent to it